### PR TITLE
[WIP] Remove ConfigService.ts

### PR DESCRIPTION
**Pull request by timgerstel**
## Proposed changes
Previously, there was two services performing the identical task of maintaining the Zowe yaml config. The original ConfigurationStore.ts performed all the operations necessary for Zen, and then ConfigService.ts was added after.

The code base then became scattered with references to two different yaml objects from the two separate services, both of which were being updated independently each time. In short, it was a mess.

This PR removes ConfigService.ts.

This PR depends upon the following PRs:

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Change in a documentation
- [X] Refactor the code 
- [ ] Chore, repository cleanup, updates the dependencies.
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## PR Checklist
Please delete options that are not relevant.
- [ ] If the changes in this PR are meant for the next release / mainline, this PR targets a "staging" branch.
- [ ] My code follows the style guidelines of this project (see: [Contributing guideline](https://github.com/zowe/zen/blob/v2.x/staging/CONTRIBUTING.md))
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] video or image is included if visual changes are made

- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works, or describe a test method below

### CODE REVIEW

File: src/renderer/components/common/EditorDialog.tsx [hunk 1]
```diff
@@ -34,24 +33,25 @@ const EditorDialog = ({contentType, isEditorVisible, toggleEditorVisibility, onC
...
   const [editorContent, setEditorContent] = useState(content ? content : '');
   const [isSchemaValid, setIsSchemaValid] = useState(true);
   const [schemaError, setSchemaError] = useState('');
   const fileInputRef = useRef(null);
   let initZoweConfig: any;
 
-  if(contentType == 'yaml') {
-    initZoweConfig = getZoweConfig();
-  }
+
```
Comment (DivergentEuropeans) [inline src/renderer/components/common/EditorDialog.tsx:44]: Can you add a comment why you did this/left it commented

File: src/renderer/components/common/EditorDialog.tsx [hunk 2]
```diff
@@ -74,15 +74,15 @@ const EditorDialog = ({contentType, isEditorVisible, toggleEditorVisibility, onC
     }
 
     if(newCode && (newCode == "\n" || newCode == "")) {
-      setZoweConfig("");
+      // setZoweConfig("");
```
Comment (DivergentEuropeans) [inline src/renderer/components/common/EditorDialog.tsx:77]: Can you add a comment why you did this/left it commented

**Review by DivergentEuropeans — APPROVED** [review]

File: src/renderer/components/common/EditorDialog.tsx (hunk 1 above)
Comment (timgerstel) [reply src/renderer/components/common/EditorDialog.tsx:44]: initZoweConfig was never called anywhere, its just dead code from staging that should be 100% removed

File: src/renderer/components/common/EditorDialog.tsx (hunk 2 above)
Comment (timgerstel) [reply src/renderer/components/common/EditorDialog.tsx:77]: Dead logic from staging that should be 100% removed

**Pull request merged**